contracts/interfaces/IForkingManager.sol

```diff
@@ -4,6 +4,46 @@ pragma solidity ^0.8.20;
 import {IForkableStructure} from "./IForkableStructure.sol";
 
 interface IForkingManager is IForkableStructure {
+    // Dispute contract and call to identify the dispute
+    // that will be used to initiate/justify the fork
+    struct DisputeData {
+        address disputeContract;
+        bytes disputeCall;
+    }
+
+    // Struct containing the addresses of the new implementations
+    struct NewImplementations {
+        address bridgeImplementation;
+        address zkEVMImplementation;
+        address forkonomicTokenImplementation;
+        address forkingManagerImplementation;
+        address globalExitRootImplementation;
+        address verifier;
+        uint64 forkID;
+    }
+
+    // Struct that holds an address pair used to store the new child contracts
+    struct AddressPair {
+        address one;
+        address two;
+    }
+
+    // Struct containing the addresses of the new instances
+    struct NewInstances {
+        AddressPair forkingManager;
+        AddressPair bridge;
+        AddressPair zkEVM;
+        AddressPair forkonomicToken;
+        AddressPair globalExitRoot;
+    }
+
+    // Struct containing the data for the paid fork
+    struct ForkProposal {
```

Comment:
Does this still need to be a struct now that there's only one of them? I think it's simpler for external contracts if they just have regular public variables, eg you can call `forkingManager.executionTime()`